pageserver/src/tenant.rs

```diff
@@ -226,7 +226,11 @@ pub(crate) struct TenantPreload {
 /// When we spawn a tenant, there is a special mode for tenant creation that
 /// avoids trying to read anything from remote storage.
 pub(crate) enum SpawnMode {
-    Normal,
+    /// Activate as soon as possible
+    Eager,
+    /// On-demand activation possibly before the tenants turn
```

Comment:
what does "before the tenants turn" mean?

Reply:
I was alluding to the queueing on the semaphore as the "tenants turn", any ideas how to make this better?

Reply:
```suggestion
    /// Activation lazily in the background, with the option to skip the queue if the need comes up
```